### ISSUE TITLE
404 error on NLM works

### ISSUE THREAD

**Issue by eroux** [buda-base/buda-iiif-server#29]
the following URL gives a 404:

http://iiif.bdrc.io/bdr:V1NLM7_I1NLM7_001::I1NLM7_0010003.jpg/full/max/0/default.jpg

but the s3 key exists though:

```
s3://archive.tbrc.org/Works/ba/W1NLM7/images/W1NLM7-I1NLM7_001/I1NLM7_0010003.jpg
```

**Comment by eroux** [comment]
(this is probably a relatively easy fix, it's probably worth doing it soonish, it's for a partnership)

**Comment by MarcAgate** [comment]
Fixed as of commit 30a75b3

**Comment by eroux** [comment]
Thanks!
